lib/backend-api/src/client.rs

```diff
@@ -75,10 +75,10 @@ impl WasmerClient {
         })
     }
 
-    pub fn new_with_proxy(
```

Comment:
To not do breaking api, can we call this `new_with_proxy` and then have a `new` that just calls `new_with_proxy`?

Reply:
Yeah makes sense.